RecursiveCraftIntegration.cs

```diff
@@ -67,18 +67,18 @@ private static void StrongRef_Unload()
 			OnPlayer.QuickSpawnItem_int_int -= OnPlayerOnQuickSpawnItem_int_int;
 		}
 
-		private static void OnPlayerOnQuickSpawnItem_int_int(OnPlayer.orig_QuickSpawnItem_int_int orig, Player self, int type, int stack)
+		private static int OnPlayerOnQuickSpawnItem_int_int(OnPlayer.orig_QuickSpawnItem_int_int orig, Player self, int type, int stack)
 		{
 			if (CraftingGUI.compoundCrafting)
 			{
 				Item item = new();
 				item.SetDefaults(type);
 				item.stack = stack;
 				CraftingGUI.compoundCraftSurplus.Add(item);
-				return;
+				return item.whoAmI;
```

Comment:
There is no point in returning anything other than 0 since we're not using the return value